File: LLDs/DocumentSerivce/cpp/model/Document.cpp
#include <bits/stdc++.h>

using namespace std;

class Document
{
private:
    string name, data, owner;

public:
    int id;

    Document(string docName, string docData, string owner) {
        this->name = docName;
        this->data = docData;
        this->owner = owner;
    }
    string getDocData() {
        return this->data;
    }

    string getDocName() {
        return this->name;
    }

    string getDocOwner() {
        return this->owner;
    }

    void setDocId(int id) {
        this->id = id;
    }
    // void setDocId(int id);
    // string getDocName();
    // string getDocData();
    // string getDocOwner();
};

File: LLDs/DocumentSerivce/Model/Document.java
package Model;
public class Document {
    private String name;
    private String data;
    private String owner;
    public Integer id;

    public Document(String docName, String docData, String username) {
        this.name = docName;
        this.data = docData;
        this.owner = username;
    }

    public String getOwner() {
        return this.owner;
    }

    public String getDocName() {
        return this.name;
    }

    public String getDocData() {
        return this.data;
    }

    public void updateDocData(String updatedDocumentData) {
        this.data = updatedDocumentData;
    }

    public void setDocumentId(int id) {
        this.id = id;
    }

}